doc/openshift-guide.md

```diff
@@ -0,0 +1,205 @@
+# Using 3scale API Gateway on Red Hat OpenShift
+
+This tutorial describes how to use the Dockerized version of the 3scale API Gateway that is packaged for easy installation and operation on Red Hat OpenShift V3.
+
+In this tutorial we use the OpenShift Origin VM &ndash; a Virtual Machine image you can download and run locally, so that you can follow this tutorial without running a full OpenShift deployment.
+
+## Tutorial Prerequisites
+
+To follow the tutorial steps below, you'll first need to address the following prerequisites:
+
+### 3scale account configuration
+
+These steps will guide you to finding information you will need later to complete the tutorial steps: your _3scale Admin URL_, your _Provider Key_, and the _Private Base URL_ of your API.
```

Comment:
I kept it like this, because it was well explained  in the article where to get the key. We can do another iteration later, and replace it with Access Token.